crates/autopilot/src/lib.rs

```diff
@@ -365,11 +365,16 @@ pub async fn main(args: arguments::Arguments) {
         other => Some(other.unwrap()),
     };
 
+    let simulation_web3 = args.simulation_node_url.as_ref().map(|node_url| {
+        shared::ethrpc::web3(&args.shared.ethrpc, &http_factory, node_url, "simulation")
+    });
```

Comment:
Should we do the same for trace web3? My rationale for not defaulting is to avoid accidentally setting `Web3` trade simulations (or `TraceCall` based bad token detection) without having a node that supports it.

Reply:
That's a good point. Maybe we shouldn't.